src/multicloud-connector/azext_multicloud_connector/aaz/latest/arc_multicloud/_generate_aws_template.py

```diff
@@ -0,0 +1,200 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "arc-multicloud generate-aws-template",
+)
+class GenerateAwsTemplate(AAZCommand):
+    """Retrieve AWS Cloud Formation template
+    """
+
+    _aaz_info = {
+        "version": "2024-08-01-preview",
```

Comment:
Will replace all the 2024-08-01-preview api-version to 2024-12-01 when the arm rollout deployment is done for all prod regions

Reply:
Updated to 2024-12-01